src/modules/FunModule.cpp

```diff
@@ -62,4 +64,36 @@ FunModule::FunModule() : Module("funutil", true), m_memeChannel(0)
 
 		});
 
+	RegisterCommand(Commands::FUN_ROLL, "roll", [this](Discord::Client& client, const Discord::Message& message, const Discord::Channel& channel)
+	{
+
+		QStringList args = message.content().split(' ');
+		GuildSetting* setting = &GuildSettings::GetGuildSetting(channel.guildId());
+		QString prefix = setting->prefix;
+
+
+		if (args.first() != prefix + "roll")
+			return;
+
+		if (args.size() != 3) {
+			client.createMessage(message.channelId(), "**Wrong Usage of Command!** ");
+			return;
+		}
+
+		int max = args.at(2).toDouble();
+		int min = args.at(1).toDouble();
```

Comment:
I would love it if you could just type `!roll 20` and it would roll 1 to 20 for you. Basically, "min" is 1 if args.size() is 2